core/lib/multivm/src/tracers/call_tracer/mod.rs

```diff
@@ -38,4 +52,24 @@ impl CallTracer {
         let cell = self.result.as_ref();
         cell.set(result).unwrap();
     }
+
+    fn push_call_and_update_stats(&mut self, call: FarcallAndNearCallCount) {
+        self.stack.push(call.clone());
+
+        self.max_stack_depth = self
+            .max_stack_depth
+            .max(self.stack.len() + call.near_calls_after);
```

Comment:
Nit: You can extract `call.near_calls_after` into a local var to avoid cloning `call` above. Cloning may be optimized away by the compiler, but I wouldn't rely on it.